### ISSUE TITLE
Hardcoded Windows paths

### ISSUE THREAD

**Issue by fkohrt** [rolkra/explore#7]
When running `explore::explore(attitude)` on a non-Windows device and clicking on _report all_, I get the following error:

```r
Warning in normalizePath(path, winslash = winslash, mustWork = mustWork) :
  path[1]="C:/R/template_report_variable.Rmd": No such file or directory
Warning: Error in abs_path: The file 'C:/R/template_report_variable.Rmd' does not exist.
  3: runApp
  2: print.shiny.appobj
  1: <Anonymous>
```

There seem to be some paths hardcoded that are specific to Windows:

https://github.com/rolkra/explore/blob/1f5d24403b757407dd541b550753629393b0a95a/R/explore.R#L1255-L1274

**Comment by rolkra** [comment]
Thanks a lot! I will fix it!

**Comment by rolkra** [comment]
A workaround for you should be:

Use
`library(explore)`
`explore(attitude)`

Instead of
`explore::explore(attitude)`

**Comment by rolkra** [comment]
Fixed version 1.0.1 is on CRAN
